index.html

```diff
@@ -1532,6 +1532,10 @@ <h3>Names and Descriptions</h3>
 the text value, language tag, and text direction, respectively.
 See <a href="#language-and-base-direction"></a> for further information.
         </p>
+        <p>
+The vocabulary required by this specification relies on RDF classes and predicates
+that cannot be localized, in order to ensure consistent semantic processing.
+        </p>
```

Comment:
It's not clear what this means. What does "cannot be localized" mean? Do you mean:

> The vocabulary used by this specification defines terms in a way that is globally unambiguous and in a way that ensures consistent semantic processing.

The only thing that throws a wrench into those words is the whole "issuer-defined" vocabulary stuff, which results in ambiguity (but, at least the ambiguity is constrained to people that don't want to define semantics for the terms they use via an inline mechanism).

Reply:
This paragraph does not make sense to me. The lexical values of RDF class and predicate URIs should be treated as opaque, i.e., without literal value in and of themselves.

Further, each RDF class and predicate URI may be associated with multiple [`schema:name`](https://schema.org/name), [`rdfs:label`](http://www.w3.org/2000/01/rdf-schema#label), [`dcterms:description`](http://purl.org/dc/terms/description), [`rdfs:comment`](http://www.w3.org/2000/01/rdf-schema#comment), [`dcterms:title`](http://purl.org/dc/terms/title), etc., values, whereby there may be a set of these covering each language of concern.

Yes, it may be challenging to get such multi-lingual properties into some externally managed vocabularies/ontologies, so this handling might be reserved for vocabularies managed by the immediate user, and they might want to set up a bunch of `{ localvocab:comment` [`skos:broader`](http://www.w3.org/2004/02/skos/core#broader) `rdfs:comment }` or `{ localvocab:comment` [`owl:equivalentProperty`](http://www.w3.org/2002/07/owl#equivalentProperty) `rdfs:comment }` or even `{ localvocab:comment` [`owl:sameAs`](http://www.w3.org/2002/07/owl#sameAs) `rdfs:comment }`, to really put their reasoner to the test.

But I do not think that "RDF classes and predicates that cannot be localized" is an accurate way of describing this (or any) vocabulary.

Reply:
What I mean is that we don't expect all terms to be localized.

For example, we don't expect `VerifiableCredential` or `VerifiablePresentation` to be localized, because doing so would break the information model.

Please feel free to suggest alternative text.

Reply:
Neither `VerifiableCredential` nor `VerifiablePresentation` is either an RDF class or an RDF predicate.

If you meant [`https://www.w3.org/2018/credentials#VerifiableCredential`](https://www.w3.org/2018/credentials#VerifiableCredential) or [`https://www.w3.org/2018/credentials#VerifiablePresentation`](https://www.w3.org/2018/credentials#VerifiablePresentation), then I say again -- these are URIs, which means that their lexical values *SHOULD* be treated as opaque.

At present, each of these is described with a single [`rdfs:label`](http://www.w3.org/2000/01/rdf-schema#label) — `"Verifiable credential"@en` and `"Verifiable presentation"@en`, respectively. These can be localized by the addition of more language-tagged literal values of that property, without breaking the information model. Optimally, such additional literal values would be added to the authoritative ontology, but they could be deployment-local additions.

What am I missing?

Reply:
@TallTed thats exactly my point... many people don't understand RDF / JSON-LD... and they might start localizing all strings... if they localize strings that become URIs that are supposed to be opaque, they will break things.